lib/01_temperature.rb

```diff
@@ -0,0 +1,7 @@
+def ftoc (fahrenheit)
```

Comment:
No space before the parenthesis `(`

```
def ftoc(fahrenheit)
```

Same with the parentheses in method definitions in the rest of this PR